### PR TITLE
Fix for issue #81

### PR DESCRIPTION
**Pull request by horheynm**
SUMMARY:
Fix for https://github.com/vllm-project/llm-compressor/issues/81

steps:
1. clone repo
2. create and activate virtual eng
3. pip3 install -e .[dev]
4. change `llm-compressor/examples/big_model_offloading/big_model_fp8.py` line 34 from 
`model_stub = "meta-llama/Meta-Llama-3-70B-Instruct"`
to
`model_stub = "microsoft/Phi-3-medium-128k-instruct"`
5. run the code with 
```python3
# determine which layers to offload to cpu based on available resources
device_map = calculate_offload_device_map(
    model_stub, reserve_for_hessians=False, num_gpus=1, torch_dtype=torch.float16, trust_remote_code=True
)
```

the output config.json will get rid of line 201 from
`    "type": "longrope"`
to
`    "type": "su"`

TEST PLAN:
Run the code to completion, make sure that the issue does not come up.

### CODE REVIEW

File: src/llmcompressor/transformers/compression/helpers.py [hunk 1]
```diff
@@ -191,6 +191,7 @@ def calculate_offload_device_map(
     model_stub: str,
     reserve_for_hessians=False,
     num_gpus: int = 1,
+    trust_remote_code: bool = True,
```
Comment (robertgshaw2-neuralmagic) [inline src/llmcompressor/transformers/compression/helpers.py:194]: this should probably be defaulted to false

**Review by Lin-K76 — APPROVED** [review]
Tested changes, fixes error regarding the rope type and the configuration python files.

**Review by Lin-K76 — APPROVED** [review]
Looks good to me!

**Review by Lin-K76 — APPROVED** [review]
Example changes look good to me

**Review by bfineran — APPROVED** [review]

**Pull request merged**